eiffel-vocabulary/EiffelActivityTriggeredEvent.md

```diff
@@ -61,20 +61,14 @@ __Description:__ Identifies a cause of the event occurring. SHOULD not be used i
 __Required:__ No  
 __Legal targets:__ [EiffelActivityTriggeredEvent](../eiffel-vocabulary/EiffelActivityTriggeredEvent.md), [EiffelTestSuiteStartedEvent](../eiffel-vocabulary/EiffelTestSuiteStartedEvent.md)  
 __Multiple allowed:__ No  
-__Description:__ Identifies the [activity](../eiffel-syntax-and-usage/glossary.md#activity) or test suite of which this event constitutes a part.
+__Description:__ Identifies the activity or test suite of which this event constitutes a part.
```

Comment:
This file hasn't been properly updated since the PRECURSOR commit; this line is changed back and the PRECURSOR link type a few lines down has disappeared.